subscribers:
  in_topics: /d455/color/image_raw
  queue_size: 1
  camera_resolution: 388800

publishers:

  object_detector:
    topic: /detected_objects

  bounding_boxes:
    topic: /bounding_boxes

  detection_image:
    topic: /detection_image

image_view:
  enable_opencv: false
